app/src/main/java/com/alphawallet/app/repository/TokenRepository.java

```diff
@@ -1222,7 +1227,25 @@ private String callCustomNetSmartContractFunction(
                     getName(address, network),
                     getContractData(network, address, stringParam("symbol"), ""),
                     getDecimals(address, network),
-                    true, chainId);
+                    true,
+                    chainId);
+        });
```

Comment:
Remove setupTokensFromLocal.

I will further clarify the flow in a longer comment at the end of the review.

Reply:
removed